tests/commands_test.py

```diff
@@ -1,7 +1,9 @@
 import asyncio
+from typing import Any, List
 from unittest import mock
 from unittest.mock import MagicMock
 
+import pylibmc  # type: ignore
```

Comment:
Can we fix the type error here?

Reply:
hmm, removed, lets c what happens, I already forgot the reason I added that

Reply:
Add to the bottom of the `.mypy.ini` file, rather than ignoring here.